src/user_interface/settings_page.py

```diff
@@ -8,13 +8,16 @@
 
 from src.user_interface.DragButton import DragButton
 from src.user_interface.pianoKey import pianoKey
+from src.user_interface.ui_comm import UICommSystem
+from src.communication.messages import Message, MessageType
 
 
 class SettingsPage(QWidget):
 
-    def __init__(self):
+    def __init__(self, comm_system):
         super().__init__()
 
+        self.comm_system = comm_system
```

Comment:
Yeah right here just do 
`self.comm_system = UICommSystem()`
and it should be good